staging/src/k8s.io/client-go/dynamic/dynamicinformer/informer.go

```diff
@@ -59,12 +44,95 @@ type dynamicSharedInformerFactory struct {
 	informers map[schema.GroupVersionResource]informers.GenericInformer
 	// startedInformers is used for tracking which informers have been started.
 	// This allows Start() to be called multiple times safely.
-	startedInformers map[schema.GroupVersionResource]bool
-	tweakListOptions TweakListOptionsFunc
+	startedInformers   map[schema.GroupVersionResource]bool
+	stoppableInformers map[schema.GroupVersionResource]cache.DoneChannel
```

Comment:
Just a question, the `startedInformers` and `stoppableInformers` always hold the same GVR set, right?

Reply:
Yep. This is a vestige from when I had separate `Start` and `StartWithStopOptions` methods. Changed so that `startedInformers` is a map to the done channel and removed `stoppableInformers`